cmd/meroxa/root/apps/deploy.go

```diff
@@ -540,12 +542,13 @@ func (d *Deploy) checkResourceAvailability(ctx context.Context) error {
 
 	d.logger.StartSpinner("\t", resourceCheckMessage)
 
+	var resources []turbineGo.Resource
 	var resourceNames []string
 	var err error
 
 	switch d.lang {
 	case GoLang:
-		resourceNames, err = turbineGo.GetResourceNames(ctx, d.logger, d.path, d.appName)
+		resources, err = turbineGo.GetResources(ctx, d.logger, d.path, d.appName)
 	case JavaScript:
 		resourceNames, err = turbineJS.GetResourceNames(ctx, d.logger, d.path, d.appName)
```

Comment:
After Anna's merge this should be returning resources as well, same for the Python instance.